### PR TITLE
Bluetooth: controller: Fix LLCP pause logic

### PR DESCRIPTION
**Pull request by thoh-ot**
This PR contains fixes for both:
- Procedure pause logic
- TX data pause logic
- RX data pause

### CODE REVIEW

File: subsys/bluetooth/controller/ll_sw/ull_adv.c [hunk 1]
```diff
@@ -1121,6 +1121,10 @@ uint8_t ll_adv_enable(uint8_t enable)
 		conn->phy_pref_rx = ull_conn_default_phy_rx_get();
 #endif /* CONFIG_BT_CTLR_PHY */
 
+#if defined(CONFIG_BT_CTLR_LE_ENC)
```
Comment (erbr-ot) [inline subsys/bluetooth/controller/ll_sw/ull_adv.c:1124]: Good idea:-)

**Comment by thoh-ot** [comment]
Adding DNM as  iv'e found some issue with this PR

**Comment by thoh-ot** [comment]
Push from yesterday have fixed the issue i saw, removing the DNM

**Review by kruithofa — APPROVED** [review]

**Review by erbr-ot — APPROVED** [review]
LBTM

**Review by cvinayak — APPROVED** [review]
quick-ack

**Pull request merged**